internal/controllers/clusterextension_controller_test.go

```diff
@@ -148,16 +148,14 @@ func TestClusterExtensionResolutionSuccessfulUnpackFails(t *testing.T) {
 			Image:   "quay.io/operatorhubio/prometheus@fake1.0.0",
 		}, &v, nil, nil
 	})
-	res, err := reconciler.Reconcile(ctx, ctrl.Request{NamespacedName: extKey})
+	res, _ := reconciler.Reconcile(ctx, ctrl.Request{NamespacedName: extKey})
 	require.Equal(t, ctrl.Result{}, res)
-	require.Error(t, err)
```

Comment:
Why are we removing the error check here? I would expect if the unpack process fails and returns an error then reconcile would return an error.

Reply:
Thought it was a lint error when in fact it was a rebase error. But fixed now.